generic/pg-goals.el

```diff
@@ -107,8 +164,9 @@ so the response buffer should not be cleared."
       (bufhist-checkpoint-and-erase))
 
     ;; Only display if string is non-empty.
-    (unless (string-equal string "")
-      (funcall pg-insert-text-function string))
+    (when (not (string-equal string ""))
+      (funcall pg-insert-text-function string)
+      (coq-parse-goals-info string))
```

Comment:
I am pretty sure that calling a Coq specific function here will break any other proof assistant.